clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,30 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model http://www.javadoc.io/doc/io.fabric8/kubernetes-model/1.0.73
+//which will map to Kubernetes 1.5 API version
+class KubernetesPodSpecDescription {
+  String name
+  String kind
```

Comment:
The java API may do extra transformations, or include fields for tracking purposes that we don't want to propagate.